dds_web/templates/mail/authenticate.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'mail/mail_base.html' %}
+{% block body %}
+<table role="presentation" border="0" cellpadding="0" cellspacing="0"
+    style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+    <tr>
+        <td>
+            <img src="cid:Logo" alt="Logo" border="0"
+                style="border:0; outline:none; text-decoration:none; display:block; Margin-bottom: 15px; max-width: 400px; max-height: 87px;">
+        </td>
+    </tr>
+    <tr>
+        <td style=" font-family: arial, sans-serif; font-size: 14px; vertical-align: top;">
+            <p
+                style="font-family: arial, sans-serif; font-size: 14px; font-weight: bold; margin: 0; Margin-bottom: 5px;">
+                Here is the requested authentication code for SciLifeLab Data Delivery System (DDS)
```

Comment:
I just noticed that _requested_ is in here as well. Let's change it to **issued**. It is in authenticate.txt as well.